docs/guide/tech-behind-calypso.md

```diff
@@ -97,8 +97,11 @@ The way we use Git with Calypso is described in the [Git Workflow document](../g
 
 - [page.js](http://visionmedia.github.io/page.js/) – router
 - [express.js](http://expressjs.com) – light server-side framework we use to serve the initial page
-- [lodash](https://lodash.com) – general purpose utility library; includes a ton of useful functions for dealing with arrays, objects, and collections
 - [webpack](http://webpack.github.io) – building a JavaScript bundle of all of our modules and making sure loading them works just fine
 - [Babel](https://babeljs.io) – for transpiling ES2015+ and JSX
 
+## Other technologies used in Calypso that are now deprecated
+
+- [lodash](https://lodash.com) – general purpose utility library; includes a ton of useful functions for dealing with arrays, objects, and collections; deprecated because of its lack of modularity and tendency to introduce code smells by implicitly converting different value types and handling nil (`null` or `undefined`) values.
```

Comment:
The most widely accepted terminology for  a value that is either `null` or `undefined` appears to be `nullish` these days, rather than `nil`. It's encoded into ES itself through the naming of `??`, i.e. the "nullish coalescing operator".

Reply:
Agreed 👍  Changed in https://github.com/Automattic/wp-calypso/pull/50369/commits/aab3610f0dfdce92637142f4347fd99046cdc8f7